overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -49315,7 +49321,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Corruption",
-          "desc:8": "Transform an Awakened Draconium Block in the world into a Corrupted Draconium Block using your Philosopher\u0027s Stone.\n\nYou can setup automation for this using Block Placers and Breakers, Annihilation and Formation Planes, or World Block Importers and Exporters; and a Mechanical User or Player Simulator with the Philosopher\u0027s Stone."
+          "desc:8": "Transform an Awakened Draconium Block in the world into a Corrupted Draconium Block using your Philosopher\u0027s Stone.\n\nOnce you make a second Philosopher\u0027s Stone, setup automation for this using Block Placers and Breakers, Annihilation and Formation Planes, or World Block Importers and Exporters; and a Mechanical User or Player Simulator."
```

Comment:
Maybe specify that its optional to have a 2nd philosophers stone, you definitely do not need to craft another one